src/AccessibilityInsights.SharedUx/Controls/PropertyInfoControl.xaml.cs

```diff
@@ -231,7 +231,7 @@ private void UserControl_Loaded(object sender, RoutedEventArgs e)
         /// </summary>
         /// <param name="sender"></param>
         /// <param name="e"></param>
-        private void lvProperties_KeyUp(object sender, KeyEventArgs e)
+        private void lvProperties_KeyDown(object sender, KeyEventArgs e)
```

Comment:
@jalkire, there's currently no KeyUp or KeyDown method in PropertyInfoControl.xaml. Should there be?